doc/api/process.md

```diff
@@ -1103,6 +1103,27 @@ and [Cluster][] documentation), the `process.connected` property will return
 Once `process.connected` is `false`, it is no longer possible to send messages
 over the IPC channel using `process.send()`.
 
+## `process.constrainedMemory()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* {number}
```

Comment:
This probably needs to be updated now that it optionally returns `undefined`.

Reply:
Fixed, thanks.